src/utils/viewer/viewer.cpp

```diff
@@ -974,12 +974,15 @@ void Viewer::mouseReleaseEvent(MouseEvent& event) {
 }
 
 void Viewer::mouseScrollEvent(MouseScrollEvent& event) {
-  if (!event.offset().y()) {
+  // shift+scroll is forced into x direction on mac, seemingly at OS level, so
+  // use both x and y offsets.
+  float scrollModVal = event.offset().y() + event.offset().x();
+  if (!(scrollModVal)) {
```

Comment:
As we discussed on Slack -- I'd rather want to see this fixed / worked around / configured at some upper level than in the application code...

Reply:
The question remains, though, should we intercept it at a lower level? I would imagine folks might want to see/use this behavior in other UI applications (if any existed -shrug-)

Reply:
I'm not a chronic mac user, and I don't know if we would ever implement something in sim/viewer that would take advantage of horizontal scrolling explicitly, so I figured this was the route of least commitment :)

Reply:
> intercept it at a lower level?

Yeah, depends. If I would patch this in Magnum, I would add some window flag to the Application to toggle this behavior (I'm neither a fan of Apple forcing this to everyone nor a library reverting that behavior unconditionally).

I would also need to check how this behaves in SDL, if it's already accounted for in some way or not.